packages/fullstack/src/adapters/mod.rs

```diff
@@ -65,6 +67,21 @@ impl ServerFnHandler {
     }
 }
 
+#[cfg(not(target_arch = "wasm32"))]
+macro_rules! spawn_platform {
```

Comment:
Can this be a function instead of a macro? Macros are a bit harder to follow and end up generating more code

Reply:
I tried, but I could not describe it in a function, not in a macro. If you can rewrite it for me, you are welcome to do so.

Reply:
This was solved without the use of macros.